docs/reference/mapping/params/copy-to.asciidoc

```diff
@@ -68,12 +68,58 @@ Some important points:
 `field_1` to `field_2` and `copy_to` on `field_2` to `field_3` expecting 
 indexing into `field_1` will eventuate in `field_3`, instead use copy_to 
 directly to multiple fields from the originating field.
+
+NOTE: `copy_to` is _not_ supported for field types where values take the form of objects, e.g. `date_range`
+
+[float]
+[[copy-to-dynamic-mapping]]
+==== Dynamic mapping
+
+Here are some important points to consider when using `copy_to` with dynamic mappings:
+
 * If the target field does not exist in the index mappings, the usual
 <<dynamic-mapping,dynamic mapping>> behavior applies. By default, with
 <<dynamic,`dynamic`>> set to `true`, a non-existent target field will be
-dynamically added to the index mappings. If `dynamic` is set to `false`, the
+dynamically added to the index mappings.
+* If `dynamic` is set to `false`, the
 target field will not be added to the index mappings, and the value will not be
-copied. If `dynamic` is set to `strict`, copying to a non-existent field will
+copied.
+* If `dynamic` is set to `strict`, copying to a non-existent field will
 result in an error.
++
+** `copy_to` fields must specify the full path if the target field is _nested_.
```

Comment:
again, preference is no italics. would also swap this around to make people spot `nested` first

```suggestion
**  If the target field is nested, then `copy_to` fields must specify the full path to the nested field.
```